### PR TITLE
Extrude draw param for polygon style

### PR DESCRIPTION
**Pull request by tallytalwar**

### CODE REVIEW

**Comment by blair1618** [comment]
Nice and simple :)

It's unclear what type `extrude` should have, though. From the tangram source it looks like it can be a boolean, a number, a function, or a pair of numbers? (https://github.com/tangrams/tangram/blob/master/src/styles/polygons/polygons.js#L61)

**Comment by tallytalwar** [comment]
Yes, I saw that.

But wanted to have something out there. And was not sure of few things like the default value.
Also it seems we are not using min_height in our extrusion code, which is used in the "pair" usecase for extrude param.

**Comment by tallytalwar** [comment]
Plus it should not be hard to do using variant for extrude parameter.

File: core/src/scene/drawRule.cpp [hunk 1]
```diff
@@ -170,6 +196,17 @@ bool DrawRule::getValue(StyleParamKey _key, int32_t& _value) const {
     return true;
 }
 
+bool DrawRule::getValue(StyleParamKey _key, std::pair<float, float>& _value) const {
+    auto& param = findParameter(_key);
+    if (!param) { return false; }
+    if (!param.value.is<std::pair<float, float>>()) {
+        logMsg("Error: not a bool\n");
```
Comment (blair1618) [inline core/src/scene/drawRule.cpp]: should say 'not a pair', probably

Comment (tallytalwar) [inline core/src/scene/drawRule.cpp]: fixed.

**Pull request merged**